public/walkthroughs/1/walkthrough.adoc

```diff
@@ -0,0 +1,318 @@
+
+[id='creating-fuse-integration_{context}']
+[.integr8ly-docs-header]
+=== Creating an integration
+
+To use the Fuse connections, you create an integration in Red Hat Fuse Online.
+
+. Select *Integrations* from the left hand menu.
+
+. Select the *Create Integration* button to start the *Create Integration* wizard.
+
+. Choose *Walkthrough {walkthrough-id} Messaging App* as the *Start Connection*.
+
+. When prompted to *Choose an Action*, select *Subscribe for messages*.
+
+. When prompted for a *Destination Name*, enter:
++
+[subs="attributes+"]
+---- 
+work-queue/requests
+----
+
+. Choose *Queue* as the *Destination Type* and click *Next*.
+
+. When prompted to *Specify Output Data Type*:
+.. Select *JSON Schema* as the type.
+.. Enter the following in the *Definition* field:
++
+[subs="attributes+"]
+----
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"type": "object",
+	"properties": {
+		"type": {
+			"type": "string"
+		}
+	}
+}
+----
+.. Click *Done*.
+
+. Choose *Walkthrough {walkthrough-id} CRUD App* as the *Finish Connection*.
+
+. When prompted to *Choose an Action*, select *Create a fruit*.
+
+. When prompted to *Add to Integration*, select *Add a Step*.
+
+. Select *Data Mapper* to map source and target fields in the corresponding JSON schemas:
+.. Click and drag *type* from the *Source* column to *body/name* in the *Target* column.
+.. Click *Done* to navigate back to the Integration screen.
+
+. Click *Publish*.
+. When prompted, enter a name, for example:
++
+[subs="attributes+"]
+----  
+Walkthrough {walkthrough-id}
+----
+
+[type=verification]
+Check the *Integrations* screen of the link:{fuse-url}[Red Hat Fuse Online, window="_blank"] console to make sure the new connection exists.
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above.  If you are still having issues, contact your administrator.
+// end::task-creating-fuse-integration[]
+
+[time=5]
+[id='using-the-application-integration']
+== Using the application integration
+:context: using-the-application-integration
+
+// tag::task-using-integration[]
+[id='using-integration_{context}']
+[.integr8ly-docs-header]
+=== Using the application integration
+
+After setting up the integration between the Node.js and Spring Boot applications, we can use the integration to create a new fruit for the grocery inventory application.
+
+:node-url: http://frontend-node-app.apps.city.openshiftworkshop.com/
+:spring-url: http://spring-boot-rest-http-crud-spring-app.apps.city.openshiftworkshop.com/
+:fuse-url: https://eval.apps.city.openshiftworkshop.com/
+
+. Navigate to the link:http://{node-js-url}[Node.js app, window="_blank"].
+
+. Enter a value for *Fruit*, for example:
++
+----  
+Pineapple
+----
+
+. Click *Send Request*.
+
+. Navigate to the link:http://{spring-boot-url}[Spring Boot app, window="_blank"].
+
+. Check that the entry from the Node.js app is displayed.
+
+
+[type=verification]
+View the activity log:
```

Comment:
@aidenkeating  this verification doesn't render like I expect, are we supporting multiline verificiations?

Reply:
@finp It should support multi-line verifications, but not multi-block verifications. Because we wouldn't know which block is the final block for the verification. Does `+` on the newline resolve the issue?

Reply:
@aidenkeating gotya, in this case a + does not work, but I can wrap the text in **** and make it a block